src/cpu/base.cc

```diff
@@ -727,6 +723,27 @@ BaseCPU::traceFunctionsInternal(Addr pc)
     }
 }
 
+void
+BaseCPU::scheduleSimpointsInstStop(std::vector<Counter> inst_starts)
+{
+    std::string cause = "simpoint starting point found";
+    for (size_t i = 0; i < inst_starts.size(); ++i){
+
+        scheduleInstStop(0, inst_starts[i], cause);
+
```

Comment:
I know I'm being nitpicky here but... can you remove the extra newlines? Also, there should be a space toward the end of the `for` line (`){` -> `) {`)